package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "reshape-standard",
   "description": "a standard plugin pack for reshape",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
So typically the maintainer will handle the version bumps, you don't need to include this in the pull request